components/ws-manager/pkg/manager/manager.go

```diff
@@ -199,7 +199,7 @@ func (m *Manager) StartWorkspace(ctx context.Context, req *api.StartWorkspaceReq
 	backoff := wait.Backoff{
 		Steps:    10,
 		Duration: 100 * time.Millisecond,
-		Factor:   5.0,
+		Factor:   2.5,
```

Comment:
Would it make sense to leverage the `cap` property for backoff? I believe it's intent is to ensure the duration does't exceed a certain time limit.

Reply:
Yeah, we could do that as well. What would be a reasonable cap? 5 min?

Reply:
I think 5 minutes will do. 

We can always bring it down if we need, but this way it's a predictable experience. 

W/o accounting for sleep (and jitter), that would get users to iteration 8 (but not beyond) here: https://go.dev/play/p/dtUxS6TwTrw